app/src/screens/SignIn/index.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { useAuth0 } from 'react-native-auth0';
+import { StyleSheet, View, Button, Image, Text } from 'react-native';
+
+const styles = StyleSheet.create({
+  container: {
+    backgroundColor: '#F4F4F6',
+    height: '100%',
+    display: 'flex',
+    alignContent: 'center',
+  },
+  helloFigureContainer: {
+    flex: 1,
+    justifyContent: 'flex-end',
+  },
+  body: {
+    marginHorizontal: 40,
+  },
+  label: {
+    fontFamily: 'Inter',
+    fontSize: 14,
+    fontWeight: '700',
+    lineHeight: 20,
+    letterSpacing: 0,
+    color: '#121C2D',
+  },
+  logoContainer: {
+    marginTop: 140,
+    marginLeft: 18,
+  },
+  text: {
+    marginBottom: 20,
+  },
+});
+
+const SignIn: React.FC = () => {
+  const { authorize } = useAuth0();
+
+  const onLogin = async () => {
+    try {
+      await authorize({ scope: 'openid profile email' });
```

Comment:
Should we put this in the config file as well?

Reply:
Do we want to allow user to config this? Without scoping an email, my code might break since an email may not be returned? 
https://auth0.com/docs/get-started/apis/scopes/openid-connect-scopes

Reply:
I think as long as we put it were we can change auth0 related things in one place, then that is good.